### PR TITLE
added riverbank class of sprites

### PR DESCRIPTION
**Pull request by WarriorAchilles**
Instantiated 6 of them to kill the player at the riverbank. Also edited the positions of the frog nests to help them match up well. One problem, it is possible for the player to collide with both the riverbank and the frog nest. Causing the player to die while also getting points.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -132,6 +140,8 @@ def main():
         spawn_water_lanes(frame_count, water_lane1, water_lane2, water_lane3, water_lane4, water_lane5,
                           render_group, asset_dict, WIN)
         draw_sprites(render_group, WIN, background)
+        # draw_sprites(win_group, WIN, background)
+        # draw_sprites(kill_group, WIN, background)
```
Comment (johnpcooke94) [inline src/main.py]: I'd prefer if you got rid of these. I'm sure you used them for figuring out the placements but I'd rather not have them there to accidentally get uncommented.

File: src/Sprites/riverbank.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import pygame.sprite
+
+
+class Riverbank(pygame.sprite.Sprite):
+    """Pygame sprite class for frog nests used for checking the win condition"""
```
Comment (johnpcooke94) [inline src/Sprites/riverbank.py]: Document ya functions bro, this is copied from the nest class 😂

**Review by johnpcooke94 — CHANGES_REQUESTED** [review]
Couple small things

File: src/main.py (hunk 1 above)
Comment (WarriorAchilles) [reply src/main.py]: Oops, yeah meant to remove those, mb.

File: src/Sprites/riverbank.py (hunk 2 above)
Comment (WarriorAchilles) [reply src/Sprites/riverbank.py]: heh...heh....guilty as charged xD

**Comment by WarriorAchilles** [comment]
changes made as requested

**Review by johnpcooke94 — APPROVED** [review]
Looks good!

**Pull request merged**